src/main/webapp/WEB-INF/tags/module/ar/paymentApplicationSummaryOfAppliedFunds.tag

```diff
@@ -0,0 +1,165 @@
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+--%>
+<%@ include file="/jsp/sys/kfsTldHeader.jsp"%>
+<%@ attribute name="hasRelatedCashControlDocument" required="true"
+	description="If has related cash control document"%>
+<%@ attribute name="readOnly" required="true"
+	description="If document is in read only mode"%>
+<%@ attribute name="isCustomerSelected" required="true"
+    description="Whether or not the customer is set" %>
+<c:set var="docHeaderAttributes" value="${DataDictionary.FinancialSystemDocumentHeader.attributes}" />
+
+<kul:tab tabTitle="Summary of Applied Funds"
+	defaultOpen="${isCustomerSelected}"
+	tabErrorKey="${KFSConstants.PAYMENT_APPLICATION_DOCUMENT_ERRORS}">
+	<div class="tab-container" align="center">
+		<c:choose>
+			<c:when test="${!isCustomerSelected}">
+		    		No Customer Selected
+	    	</c:when>
+			<c:otherwise>
+			    <h3>Summary of Applied Funds</h3>
+				<table class="standard side-margins" width="100%" cellpadding="0" cellspacing="0">
+					<tr>
+						<td style='vertical-align: top;' colspan='2'>
+							<c:choose>
+								<c:when test="${empty KualiForm.document.invoicePaidApplieds}">
+								   		No applied payments.
+							   	</c:when>
+								<c:otherwise>
+									<table width="100%" cellpadding="0" cellspacing="0" class="datatable" style="border-right: 1px solid #c3c3c3;">
+										<tr>
+											<td colspan='4' class='tab-subhead'>
+												<h4>Applied Funds</h4>
+											</td>
+										</tr>
+										<tr>
+											<th>
+												Invoice Nbr
+											</th>
+											<th>
+												Item #
+											</th>
+											<th>
+												Inv Item Desc
+											</th>
+											<th style="text-align: right">
+												Applied Amount
+											</th>
+										</tr>
+										<logic:iterate id="invoicePaidApplied" name="KualiForm"
+											property="document.invoicePaidApplieds" indexId="ctr">
+											<tr>
+												<td>
+													<c:out value="${invoicePaidApplied.financialDocumentReferenceInvoiceNumber}" />
+												</td>
+												<td>
+													<c:out value="${invoicePaidApplied.invoiceItemNumber}"/>
+												</td>
+												<td>
+													<c:out value="${invoicePaidApplied.invoiceDetail.invoiceItemDescription}" />&nbsp;
+												</td>
+												<td style="text-align: right;">
+													$<c:out value="${invoicePaidApplied.invoiceItemAppliedAmount}" />
+												</td>
+												<%-- Cornell mod is addition of this td tag --%>
+												<td>
+												    <html:html-button
```

Comment:
You should wrap this in an "if" block that checks the status of the "readOnly" attribute, to ensure that this button is hidden when the document is in a non-editable state. There should be some example areas in this file that check the "readOnly" attribute's value.

Reply:
Code adjusted.